libvcx/src/utils/constants.rs

```diff
@@ -55,6 +57,7 @@ pub const CREDENTIAL_RESPONSE: &'static [u8; 3686] = &[129, 167, 98, 117, 110, 1
+pub const GET_MESSAGES_DECRYPTED_RESPONSE: &str = r#"{"@type":"did:sov:123456789abcdefghi1234;spec/pairwise/1.0/MSGS","msgs":[{"payload":{"ciphertext":"oThWoHkEUkOvRk9ry3jjtK19DuG6e5VaXYtSrawHJJQCnfcFNwif5t6BoURbewOxQYfR7QOeJUTvEmVlVl1l2GPkNPXJDKuaqlRAMdotQVvogKAPoQY0rxloosETIkpZUc2dDeLpWO0IaSmv5Q8eeEA18CkJBfe-PsSkJna8mUVVw7w1rmSh30Dyiz2686pKBhbKcZCPHn1t6qPgQeSGPLqsXD5eJ--JAVfdQQixF5atMvkObV5ET--FnL9pUiycPC9fIXsyl_jzkJ2mhpe8SiJlEqZqBJ1DyxTWW3BRvNrP-hFx_a66-NrAEBIxctpVzRR29MI46Ipn3yJAn9zCJUOxXPL3eKB436fePuQZFDusxscYJHjtjgP7ItWylBkiCob2uleZkqprmlEnWKn_40mogkLW25LG1pkdxfMvBljRdDVvvfrHVT8AsggUJvgKmZloz4AgjQfuty_T-U83oDSR3tlHYUR6J_XTEz0w_XW_wiSwLe8rIRRvA49Ja-A6ujUtgRVIot53kh19tNiqonosH1AyS6k5ZdlPpQNnZ391B_kV4zfBdn_5vepcXjoDT9m_i67q-xK42OpKwFXT8pNnxREoOhzTa6scPxvcKdq3iDRvUIbTrt7O0-22jbmsEJ1mOQ64pitfgNKQ0jkUpFoBiDuvmlA0RtDk8azkQpBNtgx1naykrHTIfGWxD4acYUaaN9mhUIWvkzaGOE0tPT-6v_oD-mGy4eU-qaP3DNxTJRn_FMWiGmwmsWee_GUL-8qTtLAeby_J9MHJli7Wt80M0MOElCkNkW6pQuMMC9IPYadvjZeMH8NFmOh5cGRZlBGWTzlFXGFwowvBr9DrjQ2sgZB4xGkG2owdi-VB676CboGO-GdPkvXnNeubZwOVtEW1W3u5_3nHnjIEyClN7lesizpy8PUI8Ud5sqe3mZUEyO9a_ywtOFkjF8xzTyoRK-fZj6XG6qqdjUo4BpcNv9Kt56GpigEKeQQ1mQfWIJROskbe_nJ5j_vc8HxsEnrB5-Nl2xEAZFwN5CSREN5FXaPIXE1U9Xp0HQgdOZ3_9D48twqAMvV2asTjlgRAxUh8uf0mwxg=","iv":"_AmDnIVEnS2-m14K","protected":"eyJlbmMiOiJ4Y2hhY2hhMjBwb2x5MTMwNV9pZXRmIiwidHlwIjoiSldNLzEuMCIsImFsZyI6IkF1dGhjcnlwdCIsInJlY2lwaWVudHMiOlt7ImVuY3J5cHRlZF9rZXkiOiJsbzRrM0R2RzJXcnZJNTJNNTVaM2tZdXJQX0pZd2gtcUtHak5uRkE4YjZpbmJNRVBpbkUxT1Nwb2dpY2FHSGNPIiwiaGVhZGVyIjp7ImtpZCI6IjRQWGFDU1BUQVpxRFRIaDRNNnYzWlExdVc1d2hQUW5EeDMyYkp0Q3lvSjhWIiwiaXYiOiJjdUhLcUZQMlFiN0s4djJxTEFya0VLOHpRZHFkS1dLMSIsInNlbmRlciI6IkY3S2FacmhVczc0bDg2OFAzX0swYk9QVzU2eFNNc0t5M1ZPV3FGTlpsMVprOGV0UUlfZUs4Q3JXUUktT1NZU3ZlVzRHNjRZOWFnRHlDS3ZHbWN1dWtieURxOV9KVUI2Tlk1SzNIV2pjalBUVk9qZnpEX2NpV21DNTZXTT0ifX1dfQ==","tag":"GLUAUyEK7oOfB1Qyk05Ubg=="},"senderDID":"","statusCode":"MS-103","type":"aries","uid":"0c6bd83f-1fd2-441d-a0b9-3293536afdb3"}]}"#;
```

Comment:
Could you document somewhere how to reproduce this data? There's e2e encrypted data inside this mock, it would be good to know what the e2e encryption keys were

Reply:
Resolved via DM.